### DISCUSSION TITLE
Disk Utility

### DISCUSSION THREAD

**Discussion by QueenJosieBelle** [elrumo/macOS_Big_Sur_icons_replacements#1484] [Icon request] [answered]
I would absolutely love if someone could make a new Disk Utility icon that consists of either a solid red HD or SSD with a white medical cross (think the flag of Switzerland or The Red Cross logo) or a solid white HD or SSD with a red medical cross. Alternatively, a basic icon that is just solid red with a white medical cross or solid white with a red medical cross. I would greatly appreciate it, Thank you!

**Comment by vikasbhatia** [comment]
Hey @QueenJosieBelle I'll do this today. Just leaving a comment so that I don't forget.

**Comment by vikasbhatia** [comment, marked as answer]
@QueenJosieBelle Is this what you had in mind?

<img width="128" alt="Disk Utility 1" src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/11446387/24b2feba-7866-41ed-8c68-d78db6978ede">
<img width="128" alt="Disk Utility 2" src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/11446387/8647517b-0b7f-43a8-a56d-f4501c509ba8">
<img width="128" alt="Disk Utility 3" src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/11446387/9767e9ae-ecda-494e-a788-d5f3b96607fc">
<img width="128" alt="Disk Utility 4" src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/11446387/0f3ad23c-b1fd-476b-9524-9126297bbbaf">

The ICNS files will be available [on my profile](https://macosicons.com/#/u/sidewinder) once they're approved.

I'd be happy to make any edits if you need it.

**Reply by QueenJosieBelle** [reply]
PERFECT!!!!!

**Reply by QueenJosieBelle** [reply]
OMG! Thank you!!!

**Reply by vikasbhatia** [reply]
Haha you're very welcome! :)
